contracts/RiskManagerV1.sol

```diff
@@ -128,14 +128,25 @@ contract RiskManagerV1 is Auctioneer, Ownable {
             "Deposit is not in liquidation state"
         );
 
-        require(
-            deposit.collateralizationPercentage() <= collateralizationThreshold,
-            "Deposit collateralization is above the threshold level"
-        );
+        // TODO: Read ETHTBTC price ratio from ex. Uniswap. As of May 28 it is 0.0686243
```

Comment:
Uniswap will not show BTC/ETH ratio, we need some other oracle like e.g. Maker DAO's oracle used by tBTC. To fetch information from the oracle, the contract needs to be whitelisted and I think this needs to go through the governance voting so it is not that simple.

Reply:
Yes, that's why I put TBTC in a comment, not BTC. Can't we use TBTC/ETH ratio from Uniswap?

Reply:
TBTC usually trades with a premium compared to BTC given the minting costs.